### PR TITLE
Feature: Mobile menu

### PR DESCRIPTION
**Pull request by luftedar**
In this update:

- Added mobile hamburger menu.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,16 +22,30 @@
 </head>
 <body>
   <header class="d-flex d-flex-column">
+    <!-- Here Starts Mobile Menu -->
+    <nav class="mobile-menu display-none">
+      <div class="close">
+        <a href="#" id="toggle-x">X</a>
```
Comment (rloterh) [inline index.html]: The close button on the menu overlay does not have the appropriate spacing (margin/padding) to the right side, and therefore your mobile menu content does not appear centered. Kindly rectify this issue by ensuring that:
- [ ] There is **equal** spacing on the extreme left and right ends of the menu overlay, and that these spacings **scale equally**

**Your mobile menu overlay:**
image

**The original mobile menu overlay:**
image

File: index.html [hunk 2]
```diff
@@ -23,16 +22,30 @@
 </head>
 <body>
   <header class="d-flex d-flex-column">
+    <!-- Here Starts Mobile Menu -->
```
Comment (rloterh) [inline index.html]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

File: index.html [hunk 3]
```diff
@@ -23,16 +22,30 @@
...
-        <li>Contact</li>
+        <li><a href="#welcome-section">Portfolio</a></li>
+        <li><a href="#about-me">About</a></li>
+        <li><a href="#contact-me">Contact</a></li>
       </ul>
       <a href="#" id="welcome-link">
         <p id="welcome-text">Welcome</p>
       </a>
-        <img src="img/Union.png" alt="" id="toggle-menu">
+        <a class="humburger-btn"><img src="img/Union.png" alt="" id="toggle-menu"></a>
```
Comment (rloterh) [inline index.html]: - [x] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.

File: README.md [hunk 4]
```diff
@@ -2,15 +2,16 @@
```
Comment (rloterh) [inline README.md:2]: - [x] OPTIONAL
Kindly replace the text **Author1** with your full name, so as to make your ReadMe more professional. Also, endeavor to add the name(s) of any other contributor.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Orcun & Shahier, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Mobile menu is working as intended
- HTML-CSS/ JavaScript linters set up correctly
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @luftedar Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:


brent-rambo-thumbs-up

**Pull request merged**